homeassistant/components/light/hue.py

```diff
@@ -366,3 +376,15 @@ def device_state_attributes(self):
         if self.is_group:
             attributes[ATTR_IS_HUE_GROUP] = self.is_group
         return attributes
+
+    @callback
+    def _async_update_callback(self):
```

Comment:
You can remove this method and just connect the dispatcher directly to `self.async_schedule_update_ha_state`.